lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -513,10 +526,19 @@ class MockTimers {
     this.#toggleEnableTimers(true);
   }
 
+  /**
+   * The `[SymbolDispose]` method is a custom method that can be used to dispose of the MockTimers instance.
+   * It internally calls the `reset` method to reset and disable the timers.
+   * It is intended for internal use and should not be called directly in most cases.
+   */
   [SymbolDispose]() {
     this.reset();
   }
 
+  /**
+   * Resets MockTimers, disabling any enabled timers and clearing the execution queue.
+   * Does nothing if MockTimers are not enabled.
```

Comment:
maybe:
```suggestion
   * Calling it when MockTimers are not enabled will have no effect.
```

Reply:
Isn't that just a longer way to say the same thing?

Reply:
Well yes, but I think my suggestion reads clearer.
I am OK with keeping as is if you disagree 🙂